tests/test_core.py

```diff
@@ -216,9 +215,6 @@ def _validate_soundscape_and_event_audio(orig_wav_file, gen_wav_file):
 
             assert np.allclose(gen_wav, sum(gen_audio), atol=atol, rtol=rtol)
 
```

Comment:
Since we're likely to forget this (or at least me), let's add an explicit comment here that we are not comparing the original trimmed isolated events to the regenerated ones because right now `trim()` doesn't support trimming the isolated events, but that if/when we add this functionality we need to update this test as well!

Reply:
Also let's open an issue to add this functionality (trim should take an optional folder of isolated events and trim those correctly too)